### PR TITLE
Feature/proprietor account

### LINKED ISSUE #5: As a proprietor, I want to be able to create a proprietor account (epic-account)

### PR DESCRIPTION
**Pull request by btruong9**
#5 Current version of signup and login pages. Testing for signup/login validation, views, and forms in tests.py. Renders of pages are attached below. 

:sign_up:
:sign_up:proprietor
:login:
:login:proprietor:
:account:
:logout:

### CODE REVIEW

**Review by sharletclaros — DISMISSED** [review]
Nice job on the spacing, based on your demo I know everything is functional. Formatting can be handled after everyone merges. I hope this works. LGTM.

File: sadcowpng/sharedspaces/static/signup.css [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+        background-color: springgreen;
+        margin-top: 50px;
+        color: black;
+        padding: 10px 16px;
+    }
+
+    .btn:hover{
+        background-color: limegreen;
+    }
```
Comment (bishalr1) [inline sadcowpng/sharedspaces/static/signup.css:42]: newline needs to be added

File: sadcowpng/sharedspaces/templates/sharedspaces/logout.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    <title>Logout</title>
+</head>
+<body>
+
+  <p>You have successfully logged out!</p>
+  <a href="{% url 'login' %}">Click here to login again</a><br/><br/>
+  <a href="{% url 'index' %}">Home</a>
+
+</body>
+</html>
```
Comment (bishalr1) [inline sadcowpng/sharedspaces/templates/sharedspaces/logout.html:14]: newline missing

File: sadcowpng/sharedspaces/templates/sharedspaces/proprietor_login.html [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        <p>Proprietor Login</p>
+        {{ form.as_p }}
+
+    <input type="submit" value="Login">
+    <input type="hidden" name="next" value="{{ next }}">
+
+    </form>
+
+</body>
+</html>
```
Comment (bishalr1) [inline sadcowpng/sharedspaces/templates/sharedspaces/proprietor_login.html:33]: newline missing

File: sadcowpng/sharedspaces/templates/sharedspaces/proprietor_login.html [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+    <form method="post">
+    {% csrf_token %}
+        <p>Proprietor Login</p>
+        {{ form.as_p }}
+
+    <input type="submit" value="Login">
+    <input type="hidden" name="next" value="{{ next }}">
+
+    </form>
```
Comment (bishalr1) [inline sadcowpng/sharedspaces/templates/sharedspaces/proprietor_login.html:30]: Might want to indent the stuff inside form

File: sadcowpng/sharedspaces/urls.py [hunk 5]
```diff
@@ -9,4 +9,8 @@
     path('sign_up/', views.sign_up, name='sign_up'),
     path('create/', views.create_space, name='create_space'),
 
-]
+
+    path('login/proprietor/', views.ProprietorLoginView.as_view(), name='proprietor_login'),
+    path('sign_up/proprietor/', views.proprietor_sign_up_view, name='proprietor_sign_up'),
+    path('logout/', views.sign_out, name='logout'),
+]
```
Comment (bishalr1) [inline sadcowpng/sharedspaces/urls.py:18]: Another newline

File: sadcowpng/sharedspaces/templates/sharedspaces/proprietor_signup.html [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+            <table>
+                {{ form }}
+            </table>
+
+        <input style="margin-left: 50px;"  type="submit" value="Submit">
+        </form>
+    </div>
+
+</body>
+</html>
```
Comment (bishalr1) [inline sadcowpng/sharedspaces/templates/sharedspaces/proprietor_signup.html:27]: Pesky newline again

**Review by bishalr1 — DISMISSED** [review]
I found a few nit-picky things but other than that I did not see any obvious bug or required changes. I think it's good to merge once all the conflicts have been resolved.

**Review by sharletclaros — APPROVED** [review]
Spaces removed. Functionality verified. Only merging left.

**Review by bishalr1 — APPROVED** [review]
The merge conflict was resolved and it looks good. Nothing was cut out in the process. LGTM

**Pull request merged**